projects/optic/src/commands/diff/diff.ts

```diff
@@ -457,41 +459,42 @@ const getDiffAction =
       );
     }
 
-    if (options.web) {
-      if (
-        diffResult.specResults.diffs.length === 0 &&
-        (!options.check || diffResult.specResults.results.length === 0)
-      ) {
-        logger.info('Empty changelog: not opening web view');
-      } else {
-        const analyticsData: Record<string, any> = {
-          isInCi: config.isInCi,
-        };
-
-        if (!maybeChangelogUrl) {
-          const meta = {
-            createdAt: new Date(),
-            command: ['optic', ...process.argv.slice(2)].join(' '),
-            file1,
-            file2,
-            base: options.base,
+    if (config.isInCi && !options.upload)
```

Comment:
This will break the `--web` option - this will only allow `.web` to trigger in ci?